### PR TITLE
Update golang version to match what we have in the cluster-config-operator repo

### PR DESCRIPTION
**Pull request by soltysh**
/assign @tnozicka 
Same as for oc, I've manually check that these branches have 1.13, and fixed 4.4 which is actually 1.12 in the repo.

### CODE REVIEW

File: ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.12
```
Comment (tnozicka) [inline ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml:11]: isn't 4.4 1.13?

Comment (soltysh) [reply ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml:11]: nope https://github.com/openshift/cluster-policy-controller/blob/release-4.4/Dockerfile.rhel

Comment (tnozicka) [reply ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml:11]: it is https://github.com/kubernetes/kubernetes/blob/release-1.17/hack/lib/golang.sh#L470

Comment (tnozicka) [reply ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml:11]: the units are built from go 1.13 and that go 1.13 is injected into the image builds as go 1.12 so images are build with 1.13 no matter what the dockerfile says. We can't lower this to 1.12 - that would actually run the units and image builds with the wrong version - the dockefile needs to be fixed first and then the replacement here bellow - only that way we keep building with correct 1.13 all the time

Comment (soltysh) [reply ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml]: You're right, fixed in https://github.com/openshift/cluster-policy-controller/pull/31

Comment (soltysh) [reply ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml]: and here as well

**Comment by tnozicka** [comment]
/approve
/lgtm

/hold
for tests and https://github.com/openshift/cluster-policy-controller/pull/31 and https://github.com/openshift/cluster-config-operator/pull/142

File: ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.6.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
     ocp_builder_golang-1.13:
```
Comment (tnozicka) [inline ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.6.yaml:23]: I think you need to drop this automated edit `ocp_builder_golang-1.13` https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/9833/rehearse-9833-pull-ci-openshift-cluster-config-operator-master-images/1283044964091039744#1:build-log.txt%3A24

Comment (soltysh) [reply ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.6.yaml]: Good catch, thx! Should be fixed now.

**Comment by soltysh** [comment]
not needed any more
/close

**Pull request closed without merging**